website/docs/script/syscalls_for_script.mdx

```diff
@@ -48,13 +48,18 @@ A collection of CKB-VM syscalls. Also include relevant constant, such as return
 
 These are the return codes used by the CKB-VM syscalls.
 
-| Const No. | C Example              | Description                                    |
-| --------- | ---------------------- | ---------------------------------------------- |
-| 0         | CKB_SUCCESS            | No error.                                      |
-| 1         | CKB_INDEX_OUT_OF_BOUND | Index out of bound. (e.g. No such input Cell.) |
-| 2         | CKB_ITEM_MISSING       | The requested resource does not exist.         |
-| 3         | CKB_LENGTH_NOT_ENOUGH  | The supplied memory buffer too small.          |
-| 4         | CKB_INVALID_DATA       | The data provided is invalid.                  |
+| Const No. | C Example              | Description                                              |
+| --------- | ---------------------- | -------------------------------------------------------- |
+| 0         | CKB_SUCCESS            | No error.                                                |
+| 1         | CKB_INDEX_OUT_OF_BOUND | Index out of bound. (e.g. No such input Cell.)           |
+| 2         | CKB_ITEM_MISSING       | The requested resource does not exist.                   |
+| 3         | CKB_LENGTH_NOT_ENOUGH  | The supplied memory buffer too small.                    |
```

Comment:
The supplied memory buffer is too small.

Reply:
updated